velox/docs/functions/presto/conversion.rst

```diff
@@ -202,6 +220,46 @@ supported conversions to/from JSON are listed in :doc:`json`.
      -
      -
      - Y
+     - Y
+     - Y
+     - Y
+   * - tdigest
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     - Y
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+   * - qdigest
```

Comment:
Do these additions make the table too large and hard to read? If so, we may need to come up with a more efficient layout.

Reply:
Casting is to/from varbinary, if we're adding tdigest and qdigest it will have to be to row and cols

Reply:
Sure. FYI I tried compressing width by a lot and it didn't improve the issue much. (8 16 16 16 16 16 16 16 16 16 18 22 16 20 16 16 16 16 16). rST also has limitations for angled or vertical text for the column names. 

An easy follow-up idea could be increasing the width of the white part of the page